lib_comfyui/custom_extension_injector.py

```diff
@@ -0,0 +1,58 @@
+import os
+import inspect
+import ast
+import textwrap
+
+webui_custom_nodes_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), 'comfyui_custom_nodes')
+webui_custom_scripts_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), 'comfyui_custom_scripts')
+
+
+def register_webui_extensions():
+    register_custom_nodes()
+    register_custom_scripts()
+
+
+def register_custom_nodes():
+    from folder_paths import add_model_folder_path
+    add_model_folder_path('custom_nodes', webui_custom_nodes_path)
+
+
+def register_custom_scripts():
+    import server
+
+    m = ast.parse(source(server.PromptServer))
+    patch_prompt_server_init(m)
+    patch_prompt_server_add_routes(m)
+    exec(compile(m, '<string>', 'exec'), server.__dict__)
+
+
+def patch_prompt_server_init(m):

```

Comment:
I think we should consider adding a couple type annotations. Here it should be `m: ast.Module`

Reply:
https://docs.python.org/3/library/ast.html#ast.ClassDef